### DISCUSSION TITLE
절망적인 초보입니다 질문하나만 드려도 될까요?

### DISCUSSION THREAD

**Discussion by kktaxi** [bab2min/kiwipiepy#68] [Q&A]
분석하고 나온 결과 중
form 부분만 따로 추출할 수 있을까요?

아니면 form 만 나오게 설정할 수 있나요?

form 부분만 따로 빈도분석을 하고 싶은데 방법을 모르겠습니다.

**Comment by bab2min** [comment]
@kktaxi 

```python
from kiwipiepy import Kiwi
kiwi = Kiwi()
tokens = kiwi.tokenize("이것은 예시 문장이에요~")
forms = [t.form for t in tokens]
print(forms) # ['이것', '은', '예시', '문장', '이', '에요', '~']
```

빈도분석을 하고싶으시다면 python의 Counter 클래스를 이용하시면 됩니다.
```python
from collections import Counter
cnt = Counter(forms)
print(cnt.most_common())
# [('이것', 1), ('은', 1), ('예시', 1), ('문장', 1), ('이', 1), ('에요', 1), ('~', 1)]
```

**Reply by kktaxi** [reply]
정말 감사합니다. 한번 해보겠습니다.
추가: 'list' object has no attribute 'form' 라고 떠서 찾고있습니다.
 추가1:terms = []
for t in df:
    for t_i in t:
        terms.append(t_i[0]) <- 이렇게 해서 해결했습니다.


혹시 제가 지금 엑셀 파일을 빈도분석을 하려고 개발자님의 kiwi를 사용하는데 셀을 지정할고 train['내용'].apply(kiwi.tokenize) 를 사용했습니다.
여기에 Stopwords나 다른 추가 명령어를 지정할 수 있을까요? 
문과 입문자여서 문송합니다...
 추가: 해결하였습니다. 감사합니다.
